bin/NativeTests/FileLoadHelpers.cpp

```diff
@@ -67,7 +67,13 @@ HRESULT FileLoadHelpers::LoadScriptFromFile(LPCSTR filename, LPCWSTR& contents,
     //
     // Read the entire content as a binary block.
     //
-    fread((void*)contentsRaw, sizeof(char), lengthBytes, file);
+    size_t readBytes = fread((void*)contentsRaw, sizeof(char), lengthBytes, file);
+    if (readBytes < lengthBytes)
```

Comment:
I see you took this from ch/helpers. For a big file sitting on a slow drive (i.e. a drive on network) this will be most likely failing. I'm adding these both to my list. Gonna send a PR for them. For now; this one at least fails in the right place.